### PR TITLE
[action] Transfer release process from Travis to GH Actions

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    builds:
+      - default
+    wrap_in_directory: "{{ .ProjectName }}_{{ .Version }}"
+    files:
+      - README.md
+      - LICENSE
+
+source:
+  enabled: true
+  name_template: '{{ .ProjectName }}_{{ .Version }}'
```
Comment (maraino) [inline .goreleaser.yml:43]: What is the reason behind this, GitHub does this by default

File: Makefile [hunk 2]
```diff
@@ -138,200 +128,3 @@ ifneq ($(BINNAME),"")
...
-	$(call DOCKER_DEV_PUSH,autocert-controller)
-	$(call DOCKER_DEV_PUSH,autocert-init)
-	$(call DOCKER_DEV_PUSH,autocert-bootstrapper)
-	$(call DOCKER_DEV_PUSH,autocert-renewer)
-
-# starts docker registry for development
-docker-registry:
-	$Q docker run -d -p 5000:5000 --restart=always --name registry registry:2
-
-.PHONY: docker-dev docker-registry
```
Comment (maraino) [inline Makefile:230]: docker-dev goals are always useful for local development.

**Review by maraino — CHANGES_REQUESTED** [review]
Looks good, but I think we should put the docker-dev back into the makefile.

File: .goreleaser.yml (hunk 1 above)
Comment (dopey) [reply .goreleaser.yml:43]: The reason is Github could change what they do by default and we would still keep things consistent.

**Review by maraino — APPROVED** [review]
looks good

**Pull request merged**